File: python/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break


for row in range(height):
    for j in range(height - row - 1, 0, -1):
        print(" ", end="")

    for k in range(row + 1):
        print("#", end="")

    print("  ", end="")

    for l in range(row + 1):
        print("#", end="")

    print()

File: c/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int row = 0; row < height; i++)
    {
        for (int j = height - row - 1; j > 0; j--)
        {
            printf(" ");
        }
        for (int k = 0; k <= row; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int l = 0; l <= row; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}
